### PR TITLE
add a dropdown to Wallet menu #1401

### PR DESCRIPTION
**Pull request by liaojikunwork**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind backend-change
> /kind ui-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```
Screen Shot 2020-07-22 at 10 03 09 AM

### CODE REVIEW

**Comment by raullenchai** [comment]
Can we actually list them as (in this order)
```
ioPay Mobile (link to http://iopay.iotex.io/)
ioPay Desktop (link to https://iopay.iotex.io/desktop/)
ioPay Web (link to https://iotexscan.io/wallet)
```

**Comment by liaojikunwork** [comment]
So do you want to add 4 menu items under WALLET?
```
Web Wallet (link to https://iotexscan.io/wallet)
ioPay Mobile (link to http://iopay.iotex.io/)
ioPay Desktop (link to https://iopay.iotex.io/desktop/)
ioPay Web (link to https://iotexscan.io/wallet)
```

**Comment by raullenchai** [comment]
> Web Wallet (link to https://iotexscan.io/wallet)
> ioPay Mobile (link to http://iopay.iotex.io/)
> ioPay Desktop (link to https://iopay.iotex.io/desktop/)
> ioPay Web (link to https://iotexscan.io/wallet)

No, three

```
ioPay Mobile (link to http://iopay.iotex.io/)
ioPay Desktop (link to https://iopay.iotex.io/desktop/)
Web Wallet (link to https://iotexscan.io/wallet)
```

File: translations/zh-CN.yaml [hunk 1]
```diff
@@ -289,6 +289,8 @@ new-wallet.warn.do-not-lose: 不要丢失私钥！
 new-wallet.warn.do-not-share: 不要分享私钥！
 new-wallet.warn.backup: 进行备份!
 topbar.wallet: 钱包
+topbar.wallet.web: 网络钱包
```
Comment (raullenchai) [inline translations/zh-CN.yaml:292]: 网页钱包

**Comment by liaojikunwork** [comment]
Screen Shot 2020-07-22 at 10 32 01 AM

File: translations/zh-CN.yaml (hunk 1 above)
Comment (liaojikunwork) [reply translations/zh-CN.yaml]: Alright

**Pull request merged**